src/screens/SettingsScreen.js

```diff
@@ -0,0 +1,154 @@
+import PropTypes from 'prop-types';
+import React, { useContext, useState } from 'react';
+import { RefreshControl, SectionList, StyleSheet, TouchableOpacity, View } from 'react-native';
+
+import { NetworkContext } from '../NetworkProvider';
+import { OrientationContext } from '../OrientationProvider';
+import { colors, device, normalize, texts } from '../config';
+import {
+  CardListItem,
+  Icon,
+  RegularText,
+  SafeAreaViewFlex,
+  TextListItem,
+  Title,
+  TitleContainer,
+  TitleShadow,
+  Wrapper
+} from '../components';
+import { SettingsListItem } from '../components/SettingsListItem';
+import { arrowLeft } from '../icons';
+
+export const SettingsScreen = ({ navigation }) => {
+  const { isConnected, isMainserverUp } = useContext(NetworkContext);
+  const { orientation, dimensions } = useContext(OrientationContext);
+  const [refreshing, setRefreshing] = useState(false);
+
+  const keyExtractor = (item, index) => `index${index}-id${item.id}`;
+
+  const renderSectionHeader = ({ section: { title } }) =>
+    !!title && (
+      <View>
+        <TitleContainer>
+          <Title accessibilityLabel={`${title} (Überschrift)`}>{title}</Title>
+        </TitleContainer>
+        {device.platform === 'ios' && <TitleShadow />}
+      </View>
+    );
```

Comment:
this can be extracted from the component if I see it correctly

Reply:
done in https://github.com/ikuseiGmbH/smart-village-app-app/pull/123/commits/34a9b3f9ec4b76f6aecdd7429316a4c3fbab4072